index.html

```diff
@@ -1,117 +1,286 @@
 <!DOCTYPE html>
 <html>
-    <head></head>
-    <body>
-        <div>
-            <img src="https://img.hellofresh.com/image/upload/c_scale,f_auto,q_100,w_435/v1600959926/Hello_Fresh_Lockup.png">
-            <ul>
-                <li>
-                    <a href="https://www.hellofresh.com/plans" target="_blank">Our Plans</a>
-                </li>
-                <li>
-                    <a href="https://www.hellofresh.com/about/how-it-works" target="_blank">How It Works</a>
-                </li>
-                <li>
-                    <a href="https://www.hellofresh.com/tasty/gift-landing/" target="_blank">Gift Cards</a>
-                </li>
-            </ul>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
this should be in the <head>

Reply:
here's a nice resource for the `<head/>`: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head